### PR TITLE
Feature : Create model from a template

### PR DESCRIPTION
**Pull request by justinehell**

### CODE REVIEW

File: src/i18n/en-US/index.js [hunk 1]
```diff
@@ -71,6 +71,23 @@ export default {
           error: 'An error occured while deleting model.',
         },
       },
+      import: {
+        button: {
+          name: 'New model',
```
Comment (k-y0u) [inline src/i18n/en-US/index.js:76]: wrong copy paste

File: src/components/form/ImportModelTemplateForm.vue [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    @submit="onSubmit"
+    data-cy="import-model-template-form"
+    class="q-gutter-md import-model-template-form"
+  >
+    <q-input
+      v-model="modelName"
+      filled
+      :label="$t('actions.models.import.form.name')"
+      lazy-rules
+      data-cy="import-model-template-name-input"
```
Comment (k-y0u) [inline src/components/form/ImportModelTemplateForm.vue:12]: make it last attribute

File: src/components/form/ImportModelTemplateForm.vue [hunk 3]
```diff
@@ -0,0 +1,121 @@
+<template>
+  <q-form
+    @submit="onSubmit"
+    data-cy="import-model-template-form"
+    class="q-gutter-md import-model-template-form"
```
Comment (k-y0u) [inline src/components/form/ImportModelTemplateForm.vue:5]: make it first attribute

File: src/components/form/ImportModelTemplateForm.vue [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+        (v) => notEmpty($t, v),
+      ]"
+    />
+    <div class="flex row items-center justify-center">
+      <q-btn
+        icon="fa-solid fa-save"
+        :label="$t('actions.default.save')"
+        type="submit"
+        :loading="submitting"
+        data-cy="import-model-template-submit"
```
Comment (k-y0u) [inline src/components/form/ImportModelTemplateForm.vue:23]: make it last attribute

File: src/i18n/en-US/index.js (hunk 1 above)
Comment (justinehell) [reply src/i18n/en-US/index.js:76]: deleted

File: src/components/form/ImportModelTemplateForm.vue (hunk 3 above)
Comment (justinehell) [reply src/components/form/ImportModelTemplateForm.vue:5]: d

File: src/components/form/ImportModelTemplateForm.vue (hunk 2 above)
Comment (justinehell) [reply src/components/form/ImportModelTemplateForm.vue:12]: d

File: src/components/form/ImportModelTemplateForm.vue (hunk 4 above)
Comment (justinehell) [reply src/components/form/ImportModelTemplateForm.vue:23]: d

**Review by k-y0u — DISMISSED** [review]

**Review by k-y0u — APPROVED** [review]

**Pull request merged**